### PR TITLE
Spinner brandcolors

### PR DESCRIPTION
**Pull request by paulinewahle**
fix: https://github.com/Lundalogik/lime-elements/issues/3140
## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/dock/dock.scss [hunk 1]
```diff
@@ -70,7 +70,7 @@ nav {
     flex-grow: 1;
 
     padding: var(--dock-padding); // needed for the focus effect
-
+    padding-bottom: calc((var(--dock-padding) * 2.4));
```
Comment (LucyChyzhova) [inline src/components/dock/dock.scss:73]: I think this is unrelated change. Could you please remove this commit.

Comment (paulinewahle) [reply src/components/dock/dock.scss]: ⚡

**Comment by LucyChyzhova** [comment]
Nice! We should also update the `Custom colors` example, since we are saying that we use Lime Technologies' brand colors there.

**Review by LucyChyzhova — DISMISSED** [review]
Looks really nice! 😍

**Comment by Kiarokh** [comment]
really great job! 
I just noticed that there is a stroke that is constantly orange, and another stroke that keeps changing 😅 

https://github.com/user-attachments/assets/c5a12b00-7761-423c-b628-c560babeec3a

**Comment by paulinewahle** [comment]
Good catch @Kiarokh! I fixed the unbranded spinner colors. The grey brand color looks too dark in the branded spinner, I used green now but could use a non brand light grey instead if you'd like it better;)

**Comment by Kiarokh** [comment]
tnx 😊 i think a neutral color should be fine. it doesn't have to be from the brand colors. you can pick from the `--contrast-xxx` series ;)

**Comment by adrianschmidt** [comment]
> tnx 😊 i think a neutral color should be fine. it doesn't have to be from the brand colors. you can pick from the `--contrast-xxx` series ;)

+1

I was looking at the new spinner and comparing it to the old one before reading the comments here, and with green as the "neutral" colour, the spinner looks a lot "busier" than before.

(I see that you've already updated it 😊)

**Comment by adrianschmidt** [comment]
Looks good to me 🙂

https://github.com/user-attachments/assets/f932c601-b365-4288-a39d-c65033f40337

**Review by LucyChyzhova — APPROVED** [review]
Looks good! Great job!
